app/frontend/src/components/UserSummary.tsx

```diff
@@ -81,10 +81,25 @@ export default function UserSummary({
       {!user ? (
         <Skeleton />
       ) : (
-        <>{nameOnly ? user.name : `${user.name}, ${user.age}, ${user.city}`}</>
+        <>{nameOnly ? user.name : `${user.name}, ${user.age}`}
+        </>
       )}
     </Typography>
   );
+  const location = (
+    <Typography
+      color="textSecondary"
+      variant="body1"
+      noWrap={nameOnly}
+    >
+      {!user ? (
+        <Skeleton />
+      ) : (
+        <>{user.city}
+        </>
+      )}
```

Comment:
```suggestion
      ) : (
        user.city
      )}
```
Don't think you need a fragment for this since it doesn't generate an extra DOM element.